db/products.js

```diff
@@ -0,0 +1,50 @@
+const { client } = require('./index');
+
+const getProductById = async (id) => {
+    try {
+        const { rows: [product] } = await client.query(`
+           SELECT * FROM products
+           WHERE id = $1;
+        `, [id]);
+        return product;
+    } catch (error) {
+        throw error;
+    }
+}
+
+const getAllProducts = async () => {
+    try {
+        const { rows: products } = await client.query(`
+            SELECT * FROM products;
+        `);
+        return products;
+    } catch (error) {
+        throw error;
+    }
+}
+
+const createProduct = async ({
+    name,
+    description,
+    price,
+    imageURL,
+    inStock,
+    category
+}) => {
+    try {
+        const { rows: [product] } = await client.query(`
+            INSERT INTO products (name, description, price, "imageURL", "inStock", category)
```

Comment:
conditionally insert `imageURL`.  This way it can take the default.  Currently if it's not passed into the function, I believe it's inserting null.